### ISSUE TITLE
Safety nested inside the Inspection & Quality Group element

### ISSUE THREAD

**Issue by bookwolf** [theareaorg/AREA-Research-Agenda#25]
Should safety remain nested in Inspection or does it warrant its own Group?

(Please note that I also added a "taxonomy" tag so that we could track these issues together).

Who else needs to be assigned these issues? - For now I have added those of us writing topics, but please feel free to provide guidance around this and I'll adjust.

**Comment by theareaorg** [comment]
Personally, I believe that it is a good idea to break apart inspection and safety but I don't know about where quality should be. Can we separate safety as a use case without re-compiling/running all the database?

**Comment by bookwolf** [comment]
I agree that Quality could likely stay with Inspection. Not sure about impact to database, as I'm not entirely familiar with your backend system and the architecture. Perhaps that is a question for @wzbernstein ?
